pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -0,0 +1,35 @@
+/*
+ Copyright 2022 The Koordinator Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package config
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// LoadAwareSchedulingArgs holds arguments used to configure the LoadAwareScheduling plugin.
+type LoadAwareSchedulingArgs struct {
+	metav1.TypeMeta
+
+	FilterExpiredNodeMetrics    *bool                         `json:"filterExpiredNodeMetrics,omitempty"`
```

Comment:
Add a comment to each field. Just copy and paste from your proposal 😏

Reply:
done